__tests__/reductiveContext_test.re

```diff
@@ -52,7 +54,7 @@ type spyReturnMockValue = string;
 let getResult = container =>
   container->Testing.Result.result->Testing.Result.current;
 
-describe("useSelector", () => {
+describe("reductiveContext", () => {
```

Comment:
I am guessing there are so many changes in this file because of different versions of bs-platforms that format differently. Is there any way to pin the formatter version somehow, so that we get consistent formatting regardless of the bs version, @rickyvetter ?

Reply:
I think it's related to the fact that I indented everything to wrap `useSelector` and `useStore` tests in the same describe block and moved up some share code. If I try to reformat this file in master I don't see any changes.
My installed version of `bs-platform` is `5.0.6`.

Reply:
I don't know of a way to pin formatting to a specific version :\ cc @bobzhang is this something that's doable?

Reply:
I think it wasn't the problem with formatting in this case, I misunderstood it. It is just an extra level of indentation there.
We have this problem with different formatters in `reason-apollo-hooks`, so I automatically figured it must be something similar here too.